Training/2.0 Libraries.md

```diff
@@ -43,7 +49,11 @@ Create a DataFrame using the Pandas library. Your DataFrame should represent the
 
 
 ```
-
+l1 = ['A',50,'Arrhythmia']
+l2 = ['B',60,'Heart failure']
+l3 = ['C',70,'Unstable angina']
+df = pd.DataFrame(list(zip(l1, l2,l3)),
+               columns =['Patient', 'Age','Hear condition'])
 ```
 
```

Comment:
Your dataframe has linked the first element of each funciton (l1, l2, l3) by rows rather than by columns, so your table ends up something like this:
      Patient            Age   Hear condition
0           A              B                C
1          50             60               70
2  Arrhythmia  Heart failure  Unstable angina

To solve this using a similar code, you would have to group your variables by patient, age and condition rather than one for each patient:
patient = ['A','B','C']
age = [50,60,70]
condition = ['Arrythmia','Heart failure','Unstable angina']
df = pd.DataFrame(list(zip(patient, age, condition)),
               columns =['Patient', 'Age','Hear condition'])
The code returns this: 
  Patient  Age   Hear condition
0       A   50        Arrythmia
1       B   60    Heart failure
2       C   70  Unstable angina